spock-core/src/main/java/org/spockframework/runtime/SpockRuntime.java

```diff
@@ -220,6 +223,27 @@ public static Object[] despreadList(Object[] args, Object[] spreads, int[] posit
     return GroovyRuntimeUtil.despreadList(args, spreads, positions);
   }
 
+  public static final String CALL_ENTER_BLOCK = "callEnterBlock";
+  public static void callEnterBlock(SpecificationContext context, BlockInfo blockInfo) {
+    IterationInfo currentIteration = context.getCurrentIteration();
```

Comment:
How do you know that this is not nullable here?
In the other object it seams to be.